CHANGELOG.md

```diff
@@ -41,6 +42,15 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
```

Comment:
LGTM. This is where we replace `[Unreleased]` with the version like we did for v0.1.0.